### PR TITLE
Improve vim colorscheme, git integration, etc.

### PR DESCRIPTION
**Pull request by christopher-haueter**
To update your vim plugins, execute :PlugInstall in vim.

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -165,9 +165,10 @@ map <Leader>i mmgg=G`m<CR>
 syntax on
 set t_Co=256
 set background=dark
-colorscheme solarized
+colorscheme smyck
```
Comment (arktisklada) [inline vimrc]: I think this is a change that should be in your .vimrc.local file

Comment (mwenger1) [inline vimrc]: Interesting conversation.  Do you guys think there is value in everyone having the same color scheme from a consistency standpoint? Or is this the type of thing where we can each introduce a little personality to our own computers?

Comment (christopher-haueter) [inline vimrc]: Great question.

If everyone loves Solarized, then I'm happy to move it to my .vimrc.local.

If we're on Solarized b/c that came with the thoughtbot dotfiles and we haven't considered it further, might I suggest we move to Smyck? I find the color palette and contrast levels much nicer. Solarized has some really dark blues that are nearly indistinguishable from a dark terminal window.

To see what Smyck looks like, take a look at the screenshots on this page: http://color.smyck.org/

**Comment by christopher-haueter** [comment]
@arktisklada moved the smyck color scheme to my .vimrc.local. Thanks for pointing that out.

cc @mwenger1

File: vimrc.bundles [hunk 2]
```diff
@@ -43,6 +45,8 @@ Plug 'tpope/vim-repeat'
 Plug 'tpope/vim-surround'
 Plug 'vim-ruby/vim-ruby'
 Plug 'vim-scripts/tComment'
+Plug 'nomadcoder/smyck'
```
Comment (mwenger1) [inline vimrc.bundles:48]: What do you think about installing solarized as well and then everyone can easily customize or alternate:

`Plug 'altercation/vim-colors-solarized'`

Comment (christopher-haueter) [inline vimrc.bundles:48]: Done. Great idea!

Closing this PR.

**Pull request closed without merging**